### PR TITLE
Add POST method for LicenseService

### PR DESCRIPTION
**Pull request by sunithasomanna**
Added instructions for POST method for LicenseService in API readme

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -11879,15 +11879,15 @@ curl -i -X PATCH \
...
 
-| API URI                                         | Operation Applicable | Required privileges |
-| ----------------------------------------------- | -------------------- | ------------------- |
-| /redfish/v1/LicenseService                      | GET                  | `Login`             |
-| /redfish/v1/LicenseService/Licenses/            | GET                  | `Login`             |
-| /redfish/v1/LicenseService/Licenses/{LicenseID} | GET                  | `Login`             |
+| API URI                                         | Operation Applicable | Required privileges      |
+| ----------------------------------------------- | -------------------- | ------------------------ |
+| /redfish/v1/LicenseService                      | GET                  | `Login`                  |
+| /redfish/v1/LicenseService/Licenses/            | GET, POST            | `Login`, `ConfigureSelf` |
```
Comment (amar-shalgar) [inline docs/README.md]: Privilege is incorrect. Update the correct one

File: docs/README.md [hunk 2]
```diff
@@ -12015,6 +12015,61 @@ curl -i GET \
...
 
 
+## Installing a license
+
+|                    |                                                           |
+| ------------------ | --------------------------------------------------------- |
+| **Method**         | `POST`                                                    |
+| **URI**            | `/redfish/v1/LicenseService/Licenses`                     |
+| **Description**    | This endpoint installs a license on the BMC servers.      |
+| **Returns**        | License string installed on the authorized BMC server(s). |
```
Comment (amar-shalgar) [inline docs/README.md]: It should not return license string

File: docs/README.md [hunk 3]
```diff
@@ -12015,6 +12015,61 @@ curl -i GET \
...
+
+>**curl command**
+
+```
+curl -i -X POST \
+  -H "Authorization:Basic YWRtaW46T2QhbTEyJDQ=" \
+  -H "Content-Type:application/json" \
+  -d \
+'{
+  "LicenseString": "333K2-8QLV4-Y8R3G-LJQQX-7BK6M",
```
Comment (amar-shalgar) [inline docs/README.md]: Give a dummy string

File: docs/README.md [hunk 4]
```diff
@@ -12015,6 +12015,61 @@ curl -i GET \
...
+'https://{odim_host}:{port}/redfish/v1/LicenseService/Licenses'
+
+```
+
+
+>**Sample request body**
+
+```
+{
+  "LicenseString": "333K2-8QLV4-Y8R3G-LJQQX-7BK6M",
```
Comment (amar-shalgar) [inline docs/README.md]: Give a dummy string

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md]: Updated with `ConfigureManager`. Thanks for the catch

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md]: addressed comment

File: docs/README.md (hunk 4 above)
Comment (sunithasomanna) [reply docs/README.md]: done

File: docs/README.md (hunk 3 above)
Comment (sunithasomanna) [reply docs/README.md]: done

**Review by hemkapoor — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**